srcds/addons/source-python/plugins/digrats/digrats.py

```diff
@@ -840,10 +840,7 @@ def is_world_entity(index):
     if index == WORLD_ENTITY_INDEX:
         return True
 
-    if Entity(index).classname != 'player':
```

Comment:
Well, I made this one look like this because there's like a list of different conditions (together with the WORLD_ENTITY_INDEX check). If you remove this `if` and replace it with direct `return`, I'd feel like combining the first check (WORLD_ENTITY_INDEX check) there too, via `or` operator:
```python
return index == WORLD_ENTITY_INDEX or BaseEntity(index).classname != 'player'
```
Then this whole function is a one-liner.
But whatever, really, this doesn't make much difference.

Reply:
Yeah, this is probably just a matter of preferences. I usually separate multiple conditions as well, but put the last one in the return statement.